src/test/compile-fail/range_traits-1.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(inclusive_range)]
+
+use std::ops::*;
+
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
+struct AllTheRanges {
+    a: Range<usize>,
+    //~^ ERROR PartialOrd
+    //~^^ ERROR PartialOrd
+    //~^^^ ERROR Ord
+    //~^^^^ ERROR binary operation
+    //~^^^^^ ERROR binary operation
+    //~^^^^^^ ERROR binary operation
+    //~^^^^^^^ ERROR binary operation
+    //~^^^^^^^^ ERROR binary operation
+    //~^^^^^^^^^ ERROR binary operation
+    //~^^^^^^^^^^ ERROR binary operation
+    //~^^^^^^^^^^^ ERROR binary operation
```

Comment:
Will do.
On Jun 10, 2016 8:27 PM, "Brian Anderson" <notifications@github.com> wrote:

> In src/test/compile-fail/range_traits-1.rs
> <https://github.com/rust-lang/rust/pull/34180#discussion_r66696919>:
>
> > +use std::ops::*;
> > +
> > +#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
> > +struct AllTheRanges {
> > +    a: Range<usize>,
> > +    //~^ ERROR PartialOrd
> > +    //~^^ ERROR PartialOrd
> > +    //~^^^ ERROR Ord
> > +    //~^^^^ ERROR binary operation
> > +    //~^^^^^ ERROR binary operation
> > +    //~^^^^^^ ERROR binary operation
> > +    //~^^^^^^^ ERROR binary operation
> > +    //~^^^^^^^^ ERROR binary operation
> > +    //~^^^^^^^^^ ERROR binary operation
> > +    //~^^^^^^^^^^ ERROR binary operation
> > +    //~^^^^^^^^^^^ ERROR binary operation
>
> Can you file an issue about the bogus duplicate errors and add a FIXME
> here linking it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rust-lang/rust/pull/34180/files/53618c36292955b0c79e9ebc02df879015813851#r66696919>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAC3n2zMYixNPrAmRjPg75o2A7BBa3b9ks5qKgEIgaJpZM4IyD3K>
> .
>